be/src/util/hash_util.hpp

```diff
@@ -133,13 +133,21 @@ class HashUtil {
     // refer to https://github.com/apache/commons-codec/blob/master/src/main/java/org/apache/commons/codec/digest/MurmurHash3.java
     static const uint32_t MURMUR3_32_SEED = 104729;
 
+    // refer https://github.com/apache/spark/blob/v3.5.0/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/hash.scala#L615
+    static const uint32_t SPARK_MURMUR_32_SEED = 42;
```

Comment:
Do you consider the compatibility with hive hash?
Ref https://issues.apache.org/jira/browse/SPARK-17495, IMO, Hive3 use a different hash implementation from Spark's Murmur3Hash. If the bucketed table is created by HiveServer2, can we get the correct results with this PR?
Thanks.

Reply:
ok, i see you have checked the buckted table created by spark.

Reply:
yes, its specific to SPARK Distribution
Hive v1/v2 and Hudi... I shall raise a separate PR in some time(WIP)